### PR TITLE
[Usability] mandates: search by reference, add group_by, add seq type in tree view

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: account_banking_mandate/views/account_banking_mandate_view.xml [hunk 1]
```diff
@@ -68,11 +68,15 @@
 		<field name="model">account.banking.mandate</field>
 		<field name="arch" type="xml">
 			<search string="Search Banking Mandates">
```
Comment (StefanRijnhart) [inline account_banking_mandate/views/account_banking_mandate_view.xml]: Thanks for the improvements! Odd spacing here, as in the other XML. Are tabs involved, in the original or new code? Maybe this is a good time to reformat on indents of 4 spaces?

**Comment by pedrobaeza** [comment]
Can you please add also by type of mandate (CORE, B2B)?

File: account_banking_mandate/views/account_banking_mandate_view.xml (hunk 1 above)
Comment (alexis-via) [inline account_banking_mandate/views/account_banking_mandate_view.xml]: yes, it's caused by the fact that there are tabs everywhere in the XML file !!! And the lines I added have spaces.

**Comment by alexis-via** [comment]
@pedrobaeza yes, i'll update the PR with your remark when I have 5 min

**Comment by alexis-via** [comment]
This PR is finished, we can merge it. Apart from the replacement of tabs by spaces, it's just a few more lines in tree+search view of mandates. No risk.

**Comment by pedrobaeza** [comment]
All is correct as said by Alexis, so I merge. :+1:

**Pull request merged**